adapters/openx/usersync_test.go

```diff
@@ -9,7 +9,7 @@ import (
 )
 
 func TestOpenxSyncer(t *testing.T) {
-	syncURL := "https://rtb.openx.net/sync/prebid?r=localhost%2Fsetuid%3Fbidder%3Dopenx%26gdpr%3D{{.GDPR}}%26gdpr_consent%3D{{.GDPRConsent}}%26uid%3D%24%7BUID%7D"
+	syncURL := "https://rtb.openx.net/sync/prebid?gdpr={{.GDPR}}&gdpr_consent={{.GDPRConsent}}&r=localhost%2Fsetuid%3Fbidder%3Dopenx%26gdpr%3D{{.GDPR}}%26gdpr_consent%3D{{.GDPRConsent}}%26uid%3D%24%7BUID%7D"
```

Comment:
what does r=localhost mean in this context? Is this correct for general purpose?

Reply:
Oh, nevermind, this is just for tests.